docs/getting-started/install-configure.md

```diff
@@ -442,7 +442,9 @@ watch kubectl get pkg
 
 ```console
 # create service principal with Owner role
-az ad sp create-for-rbac --sdk-auth --role Owner > "creds.json"
```

Comment:
@parul5sahoo I noticed `--sdk-auth` is removed here -- does that impact the output format?

Reply:
While using the `--sdk-auth` flag I received a warning which says that the flag has been deprecated and will be removed. And yes it does impact the output format.. without the `--sdk-auth` flag these fields are present in the output, 
`"appId": " ",
  "displayName": " ",
  "password": " ",
  "tenant": " "`
And with the `--sdk-auth` flag, these fields are present 
`"clientId": "",
  "clientSecret": "",
  "subscriptionId": "",
  "tenantId": "",
  "activeDirectoryEndpointUrl": "https://login.microsoftonline.com",
  "resourceManagerEndpointUrl": "https://management.azure.com/",
  "activeDirectoryGraphResourceId": "https://graph.windows.net/",
  "sqlManagementEndpointUrl": "https://management.core.windows.net:8443/",
  "galleryEndpointUrl": "https://gallery.azure.com/",
  "managementEndpointUrl": "https://management.core.windows.net/"`
![Screenshot from 2022-06-17 20-17-54](https://user-images.githubusercontent.com/60248260/174944124-75bf663b-4585-4cb6-918f-044a6d29c4ef.png)

Reply:
Thanks for that info @parul5sahoo! Just confirming that you were in fact able to use provider-azure successfully with these changes?

Reply:
I just wanted to confirm that I can use it successfully without the `--sdk-auth`